controllers/imageupdateautomation_controller.go

```diff
@@ -65,8 +65,16 @@ import (
 	"github.com/fluxcd/image-automation-controller/pkg/update"
 )
 
-// log level for debug info
+// log level for debug output
 const debug = 1
+
+// log level for trace output; the logging system
```

Comment:
Can you create an issue for this in `fluxcd/pkg`? It will likely be useful for other controllers too now that we are standardizing helpers, and will fit well together with the user requests we have had about "debug notifications".